src/app.js

```diff
@@ -32,6 +33,6 @@
     angular.module('openfin.store', []);
     angular.module('openfin.parent', ['openfin.window']);
     angular.module('openfin.currentWindow', []);
-    angular.module('openfin.window', []);
     angular.module('openfin.scroll', []);
+    angular.module('openfin.config', []);
```

Comment:
I can remove it, but I'll need to add an exclusion into the gruntfile. If I'm doing so, I'd also like to remove `openfin.parent` since that isn't used in the child windows.